sway-fmt-v2/src/fmt.rs

```diff
@@ -680,4 +680,48 @@ trait Programmer {
             Formatter::format(&mut formatter, Arc::from(sway_code_to_format), None).unwrap();
         assert_eq!(correct_sway_code, formatted_sway_code)
     }
+    #[test]
+    fn test_impl() {
+        let sway_code_to_format = r#"script;
```

Comment:
The code to format and the corrected sway code are the same, is that intentional?

Reply:
Ah, thanks I forgot to make it messy!